tools/plugins/com.liferay.ide.upgrade.problems.core/src/com/liferay/ide/upgrade/problems/core/internal/liferay71/BREAKING_CHANGES.markdown

```diff
@@ -1045,4 +1045,28 @@ the permissions.
 
 This change removes old logic that is no longer used in Liferay Portal.
 
+---------------------------------------
+
+### Changed ArtifactId of Spring Extender [](id=changed-atifactid-of-spring-extender)
+- **Date:** 2018-Oct-04
+- **JIRA Ticket:** LPS-85710
+
+#### What changed? [](id=what-changed-25)
+
+The dependency artifactId for the spring extender dependencies have changed.
```

Comment:
Hey @jhinkey could you give a review of the documentation in this breaking change?